src/mcts/search.h

```diff
@@ -94,6 +95,7 @@ class Search {
   std::int64_t GetTotalPlayouts() const;
   // Returns the search parameters.
   const SearchParams& GetParams() const { return params_; }
+  void SendMovesStats() const;
```

Comment:
Should be moved back.

Reply:
Leftover from the stats command. Deleted.